patches/server/0920-Fix-MC-123848.patch

```diff
@@ -0,0 +1,24 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: BillyGalbreath <blake.galbreath@gmail.com>
+Date: Fri, 30 Sep 2022 16:56:07 -0500
+Subject: [PATCH] Fix MC-123848
+
+
+diff --git a/src/main/java/net/minecraft/world/entity/decoration/ItemFrame.java b/src/main/java/net/minecraft/world/entity/decoration/ItemFrame.java
+index d2a77b4ca343d19e1c70afe3f3906a9bd53d0eec..e2e9777b03bd786907ceaa7d773dd177fda28eff 100644
+--- a/src/main/java/net/minecraft/world/entity/decoration/ItemFrame.java
++++ b/src/main/java/net/minecraft/world/entity/decoration/ItemFrame.java
+@@ -282,6 +282,13 @@ public class ItemFrame extends HangingEntity {
+         }
+     }
+ 
++    // Purpur start
```

Comment:
Average hacktoberfest PR, blatantly stealing patches from other forks, thinking we wouldn't realize, smh...
```suggestion
+    // Paper start
```